cfpb/sbl-frontend
# [Filing home] routing based on user progress in the Filing process

Closes #534 

## Changes
- Filing app routing now accounts for all Submission states
- Updated page content with the latest copy
- Makes additional layout adjustments to Filing home page and InstitutionCards
- [InstitutionCards] Heading H5
- [Global] More accurate spacing - 15.008px (0.938rem) => 15px (0.9375rem)

## How to test
- [See comment below](https://github.com/cfpb/sbl-frontend/pull/535#issuecomment-2116274366).
